### ISSUE TITLE
Some samples do not work out of box

### ISSUE THREAD

**Issue by ryanbrandenburg** [aspnet/AspNetCore#7288]
At least the following samples do not work if you do `.\build.cmd` move into their directory and do `dotnet run`.

- [OpenIdConnect.AzureAdSample](https://github.com/aspnet/AspNetCore/tree/master/src/Security/Authentication/OpenIdConnect/samples/OpenIdConnect.AzureAdSample)
- [WsFederation](https://github.com/aspnet/AspNetCore/tree/master/src/Security/Authentication/WsFederation/samples/WsFedSample)
- [OpenIdConnectSample](https://github.com/aspnet/AspNetCore/tree/master/src/Security/Authentication/OpenIdConnect/samples/OpenIdConnectSample)

These samples should either be fixed so that an outside user can run them without modification (or if modification is required for config or some such, it should be explicitly called out in the error message), or removed entirely if they no longer serve a purpose.

More generally it would be a good idea for us build and test basic functionality of all our samples (IE, index page returns 200) so we can ensure that they are up to date and not misleading to customers.

**Comment by Tratcher** [comment]
I just fixed one issue with the missing launchsettings in https://github.com/aspnet/AspNetCore/pull/7105.

The other blocker here is that these scenarios require client secrets, it's a very bad practice to check those into source control. That also prevents automated testing.

Some of these samples have explicit config instructions such as https://github.com/aspnet/AspNetCore/blob/b21c09665e868526057dd3785f1d4518359b9a25/src/Security/Authentication/samples/SocialSample/Startup.cs#L51-L56

**Comment by Tratcher** [comment]
Also, these samples should stay as they make for very good manual verification of the various providers.

**Comment by ryanbrandenburg** [comment]
Yeah, if you have uses for these by all means keep them. The error message you linked is pretty good, but maybe it could call out specifically where to set that config, or link to docs on the subject?
